_layouts/newsletters.md

```diff
@@ -0,0 +1,45 @@
+---
+type: pages
+layout: default
+---
+<link rel="stylesheet" href="/assets/css/main.css">
+
+<div class="localization">
+  <a href="/en/newsletters/">en</a>
+  {% for lang in site.languages %}
+    | <a href="/{{ lang }}/newsletters/">{{lang}}</a>
+  {% endfor %}
+</div>
+
+<h1 class="post-title">Newsletters</h1>
+
+{%- if page.lang == "en" -%}
+  {% include newsletter-signup.html %}
+{%- endif -%}
```

Comment:
This is not strictly necessary, but I think you want to wrap the signup form in a div, either in this file or in `_includes/newsletter-signup.html`.  Otherwise, the raw HTML is reprocessed by Kramdon and produces some slight changes.  I'm no HTML expert, but I don't think any of those changes should produce any differences, so I'm ok with this being left as-is.

```diff
-<form action="https://bitcoinops.us18.list-manage.com/subscribe/post?u=70c3f85e5d13ffec674f30af8&amp;id=adc93c9774" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate="">
+<form action="https://bitcoinops.us18.list-manage.com/subscribe/post?u=70c3f85e5d13ffec674f30af8&amp;id=adc93c9774" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
     <div id="mc_embed_signup_scroll">
        <label for="mce-EMAIL">Subscribe to our newsletter</label>
-       <input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="email address" required="" />
+       <input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="email address" required>
     <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
-    <div class="mc-input" aria-hidden="true"><input type="text" name="b_70c3f85e5d13ffec674f30af8_adc93c9774" tabindex="-1" value="" /></div>
-    <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button" /></div>
+    <div class="mc-input" aria-hidden="true"><input type="text" name="b_70c3f85e5d13ffec674f30af8_adc93c9774" tabindex="-1" value=""></div>
+    <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
     </div>
```

Reply:
I tried adding `<div>`s but that didn't seem to prevent these changes. In any case, I've tested and the changes don't seem to have any difference in the functionality, so I'm going to leave this as-is.